doc/api/readline.md

```diff
@@ -527,11 +528,117 @@ rl.on('line', (line) => {
 });
 ```
 
+## TTY keybindings
+
+<table>
+  <tr>
+    <th>Keybindings</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>shift</code> + <code>backspace</code></td>
+    <td>Delete line left</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>shift</code> + <code>delete</code></td>
+    <td>Delete line right</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>c</code></td>
+    <td>Emit SIGINT</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>h</code></td>
+    <td>Delete left</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>d</code></td>
+    <td>Delete right or EOF</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>u</code></td>
+    <td>Delete from current to line start</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>k</code></td>
+    <td>Delete from current to end of line</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>a</code></td>
+    <td>Go to start of line</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>e</code></td>
+    <td>Go to to end of line</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>b</code></td>
+    <td>Back one character</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>f</code></td>
+    <td>Forward one character</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>l</code></td>
+    <td>Clear screen</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>n</code></td>
+    <td>Next history item</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>p</code></td>
+    <td>Previous history item</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>z</code></td>
+    <td>Moves running process into background. Type
+    <code>fg</code> and press <code>enter</code>
+    to return.</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>w</code> or <code>ctrl</code>
+    + <code>backspace</code></td>
+    <td>Delete backwards to a word boundary</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>delete</code></td>
+    <td>Delete forward to a word boundary</td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>left</code></td>
+    <td>Word left </td>
+  </tr>
+  <tr>
+    <td><code>ctrl</code> + <code>right</code></td>
+    <td>Word right</td>
+  </tr>
+  <tr>
+    <td><code>meta</code> + <code>b</code></td>
+    <td>Word left</td>
+  </tr>
+  <tr>
+    <td><code>meta</code> + <code>f</code></td>
+    <td>Word right</td>
+  </tr>
+  <tr>
+    <td><code>meta</code> + <code>d</code> or <code>meta</code>
+    + <code>delete</code></td>
+    <td>Delete word right</td>
```

Comment:
`Meta` + `Delete` does not work on Windows

Reply:
Also on WSL

Reply:
On macOS, meta+d works, but meta+delete does  a delete word left rather than right.